src/createTheme.spec.ts

```diff
@@ -16,7 +17,7 @@ describe('createTheme', () => {
         ...defaultThemeColors,
         primaryDark: 'foo',
       },
-    });
+    } as const);
 
     expect(theme.colors.primaryDark).toEqual('foo');
```

Comment:
@erikshestopal Do you get types for `theme.colors.primaryDark` ? My VsCode is treating it as `any`.

Reply:
I had to restart the TS server 
<img width="814" alt="Screen Shot 2020-07-27 at 12 08 20 PM" src="https://user-images.githubusercontent.com/6404663/88581374-e98b8100-d001-11ea-8bb4-fa80fce261d6.png">